tests/spec/w3c/headers-spec.js

```diff
@@ -136,6 +136,34 @@ describe("W3C — Headers", () => {
       expect(firstDt.textContent).toContain("Latest editor's draft:");
     });
 
+    it("prefixes status text properly", async () => {
+      const edDoc = await makeRSDoc(
+        makeStandardOps({
+          shortName: "test",
+          specStatus: "ED",
```

Comment:
Test in a for loop with some more specStatus?

Reply:
was going to, but the text is different.

Reply:
Can use `match` on text.

Reply:
Or never mind. YOLO.